src/types.rs

```diff
@@ -248,6 +248,55 @@ impl From<JsonValue> for Value {
     }
 }
 
+impl From<Value> for JsonValue {
```

Comment:
Could  you please add a docstring? (even though this is pretty self-explanatory, I admit)